stdlib/seq.ml

```diff
@@ -606,7 +616,7 @@ let rec transpose xss () =
 let rec diagonals remainders xss () =
   match xss() with
```

Comment:
If we keep the spacing change, there are a few spots (like this one) where it hasn’t been applied.

Reply:
Nice catch. Fixed.